pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java

```diff
@@ -364,6 +371,8 @@ public static class Query {
       public List<List<Object>> _outputs = null;
       @JsonProperty("expectedException")
       public String _expectedException;
+      @JsonProperty("keepOutputRowOrder")
+      public boolean _keepOutputRowOrder;
```

Comment:
good catch. would you be able to also change the OrderBy.json and add the keepOutputRowOrder flag?

Reply:
done, ran into an issue with a few tests and left TODOs to look into them later along with comments explaining what I think is going on. As a follow up perhaps I can revisit making this 'keepOutputRowOrder' less flaky in terms of ordering (i.e. handle scenarios where the order by key is correctly ordered but there are ties)